docs/API.md

```diff
@@ -683,9 +683,11 @@ Example response:
 
 #### `"params"`
 
-An object with the property:
+An object with the properties:
 
 * `"domain"`: A *domain name*, required. The domain whose DNS records are requested.
+* `"language"`: A [Language Tag], optional, used for validation error messages
+  translation, if not provided messages will be untranslated.
```

Comment:
`untranslated` -> `in English`?

There's another one like this below.

Reply:
I think it is good if it says "untranslated", but it could be "untranslated (in English)" to make it clear.

Reply:
Updated